src/main/java/com/gupb/manager/model/Student.java

```diff
@@ -73,7 +82,11 @@ public void setLastName(String lastName) {
         this.lastName = lastName;
     }
 
-    public void setIndexNumber(int indexNumber) { this.indexNumber = indexNumber; }
+    public void setIndexNumber(String indexNumber) { this.indexNumber = indexNumber; }
```

Comment:
Why index number is now String?

Reply:
In general, a number that you're not planing to do calculations on, should be stored as a form of string. If there is a possibility that the number will start with a zero, this is essential.

Reply:
Ok